accounts/accounts_test.go

```diff
@@ -33,26 +34,24 @@ func TestSign(t *testing.T) {
 	am := NewManager(ks)
 	pass := "" // not used but required by API
 	a1, err := am.NewAccount(pass)
-	toSign := randentropy.GetEntropyCSPRNG(32)
```

Comment:
Any specific reason why this is replaced with a static value? Random values can be useful to ensure signing works for arbitrary data.